modin/experimental/xgboost/test/test_xgboost.py

```diff
@@ -0,0 +1,172 @@
+                "max_depth": randint(2, 6),
+                "n_estimators": randint(100, 150),
+                "subsample": uniform(0.6, 0.4),
+            },
+        ),
+        ("load_digits", load_digits(), {"max_depth": 6, "n_estimators": 200}),
+        (
+            "load_wine",
+            load_wine(),
+            {
+                "objective": "multi:softprob",
+                "eta": 0.4,
+                "max_depth": 3,
+                "gamma": 1.0,
+                "num_class": 3,
+            },
+        ),
+        (
+            "load_breast_cancer",
+            load_breast_cancer(),
+            {"objective": "binary:logistic", "eval_metric": ["logloss", "error"]},
+        ),
+    ],
+    ids=[
+        "load_iris",
+        "load_diabetes",
+        "load_digits",
+        "load_wine",
+        "load_breast_cancer",
+    ],
+)
+def test_dmatrix_train_predict_sklearn_datasets(data, num_actors, num_round):
+    name, dataset, param = data
+
+    X = dataset.data
+    y = dataset.target
+    dmatrix_xgb = xgboost.DMatrix(X, label=y)
+    with pytest.raises(AssertionError):
+        dmatrix_mxgb = xgb.DMatrix(X, label=y)
+
+    modin_X = pd.DataFrame(X)
+    modin_y = pd.DataFrame(y)
+    dmatrix_mxgb = xgb.DMatrix(modin_X, label=modin_y)
+
+    evals_result_xgb = {}
+    evals_result_mxgb = {}
+    bst = xgboost.train(
+        param,
+        dmatrix_xgb,
+        num_round,
+        evals_result=evals_result_xgb,
+        evals=[(dmatrix_xgb, "train")],
+    )
+    modin_bst = xgb.train(
+        param,
+        dmatrix_mxgb,
+        num_round,
+        evals_result=evals_result_mxgb,
+        evals=[(dmatrix_mxgb, "train")],
+        num_actors=num_actors,
+    )
+
+    predictions = bst.predict(dmatrix_xgb)
+    modin_predictions = modin_bst.predict(dmatrix_mxgb, num_actors=num_actors)
+
+    if name in ["load_iris", "load_wine"]:
+        best_preds = np.asarray([np.argmax(line) for line in predictions])
+        modin_best_preds = np.asarray([np.argmax(line) for line in modin_predictions.T])
+
+        preds = pd.DataFrame(best_preds).apply(lambda x: round(x))
+        modin_preds = pd.DataFrame(modin_best_preds).apply(lambda x: round(x))
+
+        val = precision_score(y, preds, average="macro")
+        modin_val = precision_score(y, modin_preds, average="macro")
+
+        np.testing.assert_allclose(val, modin_val, rtol=8, atol=0.05)
+    else:
+        preds = pd.DataFrame(predictions).apply(lambda x: round(x))
+        modin_preds = modin_predictions.apply(lambda x: round(x))
+
+        val = accuracy_score(y, preds)
+        modin_val = accuracy_score(modin_y, modin_preds)
+
+        np.testing.assert_allclose(val, modin_val, rtol=0.05, atol=0.05)
+
+
+def test_error():
+    list_df = [[1, 2.0, True], [2, 3.0, False]]
+    with pytest.raises(AssertionError):
+        xgb.DMatrix(list_df, label=pd.Series([1, 2]))
+
+    param = {"a": 2, "b": 1, "c": "binary:logistic"}
```

Comment:
```suggestion
    param = {}
```
It doesn't make sense for this test

Reply:
In this line now param={}